checker/jtreg/nullness/issue1582/foo/Foo.java

```diff
@@ -0,0 +1,28 @@
+package foo;
+
+import org.checkerframework.checker.nullness.qual.EnsuresNonNullIf;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class Foo {
+
+    //  @Nullable Object theObject;
```

Comment:
I'm not sure what the commented-out lines in this file are for. Can they be removed? Or can you add a comment to document?